src/flush-microtasks.js

```diff
@@ -17,6 +19,7 @@ function getIsUsingFakeTimers() {
 
 let didWarnAboutMessageChannel = false
 let enqueueTask
+
 try {
   // read require off the module object to get around the bundlers.
   // we don't want them to detect a require and bundle a Node polyfill.
```

Comment:
Let's add `Scheduler = nodeRequire.call(module, 'scheduler')` below the `enqueueTask` assignment below here (inside the `try` block).

Then, just after the `catch` block, add this:

```javascript
function scheduleCallback(cb) {
  const NormalPriority = Scheduler
    ? Scheduler.NormalPriority || Scheduler.unstable_NormalPriority
    : null

  const scheduleFn = Scheduler
    ? Scheduler.scheduleCallback || Scheduler.unstable_scheduleCallback
    : (cb) => cb();

  return isModernScheduleCallbackSupported
    ? scheduleCb(NormalPriority, cb)
    : scheduleCb(cb)
}
```

(sorry, github won't let me make a suggestion in a place that wasn't changed).